apps/sensenet/src/components/drawer/PermanentDrawer.tsx

```diff
@@ -121,11 +129,20 @@ const PermanentDrawer: React.FunctionComponent<RouteComponentProps> = props => {
                 />
                 <ListItemSecondaryAction>
                   {device === 'mobile' ? null : (
-                    <Link to={`/personalSettings`} style={{ textDecoration: 'none' }}>
+                    <NavLink
+                      to="/personalSettings"
+                      style={{ textDecoration: 'none' }}
+                      isActive={match => {
+                        if (!match) {
```

Comment:
Please put the setCurrentPath logic to the onChange prop and remove this prop, also do this for the other navlink below.